ddp-workspace/projects/ddp-dsm-ui/src/app/shipping/shipping.component.ts

```diff
@@ -516,27 +516,44 @@ export class ShippingComponent implements OnInit {
   }
 
   private downloadKitList(fieldNames?: string[]): void {
-    const map: { realm: string; participantId: string; shortID: string; mfCode: string; sent: string; received: string }[] = [];
+    const map: { realm: string; participantId: string; shortID: string; mfCode: string; sent: string; received: string;
+      deactivated: string; deactivationReason: string; shippingId: string; type: string; sampleType: string;}[] = [];
 
     for (const kitRequest of this.kitRequests) {
       let sentDate: string = null;
-      if (kitRequest.scanDate !== 0) {
+      if (kitRequest.scanDate !== 0 && kitRequest.scanDate != null) {
```

Comment:
to avoid existing invalid date error seen in the downloaded spreadsheet if no date